pkg/edgedictionaryitem/update.go

```diff
@@ -0,0 +1,54 @@
+package edgedictionaryitem
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v2/fastly"
+)
+
+// UpdateCommand calls the Fastly API to describe a service.
+type UpdateCommand struct {
+	common.Base
+	manifest manifest.Data
+	Input    fastly.UpdateDictionaryItemInput
+
+	itemvalue common.OptionalString
+}
+
+// NewUpdateCommand returns a usable command registered under the parent.
```

Comment:
As per the following comment...
```suggestion
// NewUpdateCommand returns a usable command registered under the parent.
//
// TODO(integralist) update to not use common.OptionalString once we have a
// new Go-Fastly release that modifies UpdateDictionaryItemInput so that the 
// ItemValue is no longer optional.
```